Sources/ComposableArchitecture/Documentation.docc/Tutorials/MeetTheComposableArchitecture/02-Navigation/04-NavigationStacks/02-04-01-code-0006.swift

```diff
@@ -6,6 +6,6 @@ struct ContactDetailView: View {
   var body: some View {
     Form {
     }
-    .navigationBarTitle(Text(store.contact.name))
+    .navigationTitle(Text(store.contact.name))
```

Comment:
Switching from the deprecated [navigationBarTitle](https://developer.apple.com/documentation/swiftui/view/navigationbartitle(_:)-6p1k7) to the newer modifier.